pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -48,31 +48,21 @@ func (h *Helper) GetDockerAuth(registry, authType string) (docker.AuthConfigurat
 	}
 	if _, err := os.Stat(dockercfgPath); err != nil {
 		glog.V(3).Infof("Problem accessing %s: %v", dockercfgPath, err)
-		return authCfg, false
+		return docker.AuthConfiguration{}, false
 	}
 	cfg, err := readDockercfg(dockercfgPath)
 	if err != nil {
 		glog.Errorf("Reading %s failed: ", dockercfgPath, err)
-		return authCfg, false
+		return docker.AuthConfiguration{}, false
```

Comment:
@deads2k it is not error in context of this function as when the dockercfg does not exists or the reading of it fails it means we have no dockercfg and we skip it (and log the error to build log). We can come to review this post-3.0 (i can create follow up trello card)